flink-scala-shell/src/main/scala/org/apache/flink/api/scala/FlinkILoop.scala

```diff
@@ -124,7 +127,9 @@ class FlinkILoop(
     "org.apache.flink.api.java.operators._",
     "org.apache.flink.api.java.sampling._",
     "org.apache.flink.api.scala._",
-    "org.apache.flink.api.scala.utils._"
+    "org.apache.flink.api.scala.utils._",
+    "org.apache.flink.streaming.api.scala._",
+    "org.apache.flink.streaming.api.windowing.time._"
```

Comment:
I've added `streaming.api.scala` and `windowing.time` any other default imports important for streaming?